packages/ui/src/views/repo/repo-sidebar/index.tsx

```diff
@@ -1,11 +1,11 @@
-import { BranchListProps, BranchSelector, Button, ButtonGroup, Icon, SearchFiles } from '@/components'
+import { BranchSelector, Button, ButtonGroup, Icon, SearchFiles, type BranchSelectorItem } from '@/components'
 import { SandboxLayout } from '@/views'
 
 interface RepoSidebarProps {
   hasHeader?: boolean
   hasSubHeader?: boolean
   selectedBranch: string
-  branchList: BranchListProps[] | undefined
+  branchList: BranchSelectorItem[] | undefined
```

Comment:
Can this be made optional instead of specifying as possibly `undefined`?
```suggestion
  branchList?: BranchSelectorItem[]
```